bc3_importer/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "BC3 Connector",
+    "summary": """
+        BC3 Connector.""",
+    "author": "Binhex, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/vertical-construction",
+    "category": "Sales/Sales",
```

Comment:
As you have created a module category:

```suggestion
    "category": "BC3",
```